### PR TITLE
Deployment

### PR DESCRIPTION
**Pull request by TylerRockwell**
### Why?
https://github.com/smashingboxes/apprenticeship/tree/master/Curriculum/Backend/bookstore
Because who's going to buy from my bookstore if it's only on my machine?

### What Changed?
Set up Tape
Changed database to PG
Misc. production bugfixes

### CODE REVIEW

File: config/environments/production.rb [hunk 1]
```diff
@@ -12,4 +12,7 @@
   config.active_support.deprecation = :notify
   config.log_formatter = ::Logger::Formatter.new
   config.active_record.dump_schema_after_migration = false
+  #ignore email for now
```
Comment (BrandonMathis) [inline config/environments/production.rb]: This is a silly linting config. :-/

**Comment by BrandonMathis** [comment]
This all looks good to me  :)

File: config/environments/production.rb (hunk 1 above)
Comment (TylerRockwell) [inline config/environments/production.rb]: But how can you read my comment without the leading space? I noticed it after I made the commit, so I decided I'd just let it irritate hound.

**Comment by TylerRockwell** [comment]
Cool. Thanks, @BrandonMathis ! Btw, Tape feels like magic!

**Pull request merged**